### PR TITLE
DEVDOCS-2303: Created Widgets Versioning tutorial

### PR DESCRIPTION
**Pull request by ektravel**
# [DEVDOCS-2303](https://jira.bigcommerce.com/browse/DEVDOCS-2303)

## What changed?
* added a new tutorial
* added images

### CODE REVIEW

File: docs/api-docs/storefront/widgets/widget-versioning-tutorial.md [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+<div class="HubBlock-content">
+
+> ### Note
+> * Make a note of `uuid` and `current_version_uuid`. We will use them in the steps that follow.
+
+</div>
+</div>
+</div>
+
+You can view your widget template in the store's control panel using [Page Builder](https://support.bigcommerce.com/s/article/Page-Builder), BigCommerce's powerful theme customization tool. Your newly created widget template will be located in the left pane under **Custom**.
```
Comment (traci707) [inline docs/api-docs/storefront/widgets/widget-versioning-tutorial.md:85]: You can view your widget template in the store's control panel using [Page Builder](https://support.bigcommerce.com/s/article/Page-Builder), BigCommerce's powerful theme customization tool. You can locate your newly created widget template in the left pane under **Custom**.

File: docs/api-docs/storefront/widgets/widget-versioning-tutorial.md [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+  ],
+  "template": "<h1 style='color:red;'>Hello, World!</h1>",
+  "create_new_version": true
+}
+```
+
+[![Open in Request Runner](https://storage.googleapis.com/bigcommerce-production-dev-center/images/Open-Request-Runner.svg)](https://developer.bigcommerce.com/api-reference/store-management/widgets/widget-template/updatewidgettemplate#requestrunner)
+
+Let's look at our widget again. Send a `GET` request to [`/v3/content/widgets/{uuid}`](https://developer.bigcommerce.com/api-reference/store-management/widgets/widget/getwidget) to retrieve your widget's data.
+You will notice that `current_version_uuid` has changed and is now different from the widget's `version_uuid`. Although, we have updated the widget template, the widget created using that template did not change.
```
Comment (traci707) [inline docs/api-docs/storefront/widgets/widget-versioning-tutorial.md:212]: You will notice that `current_version_uuid` has changed and is now different from the widget's `version_uuid`. Although we have updated the widget template, the widget created using that template did not change.

File: docs/api-docs/storefront/widgets/widget-versioning-tutorial.md [hunk 3]
```diff
@@ -0,0 +1,246 @@
...
+
+## Create a new widget
+
+All widgets created after the update, will be based on the most current version. Let's create another widget to test this functionality.
```
Comment (traci707) [inline docs/api-docs/storefront/widgets/widget-versioning-tutorial.md:216]: All widgets created after the update will be based on the most current version. Let's create another widget to test this functionality.

**Review by traci707 — APPROVED** [review]

**Review by traci707 — COMMENTED** [review]
LGTM

File: docs/api-docs/storefront/widgets/widget-versioning-tutorial.md [hunk 4]
```diff
@@ -0,0 +1,257 @@
...
+* A test store.
+* API `access_token` with `content modify` scope.
+* Knowledge of the [Widgets API](https://developer.bigcommerce.com/api-docs/storefront/widgets/widgets-overview).
+
+<div class="HubBlock--callout">
+<div class="CalloutBlock--info">
+<div class="HubBlock-content">
+
+> ### Note
+> * This tutorial uses [Cornerstone](https://github.com/bigcommerce/cornerstone) theme.
```
Comment (uwikaiddi) [inline docs/api-docs/storefront/widgets/widget-versioning-tutorial.md:28]:
```suggestion
> This tutorial uses [Cornerstone](https://github.com/bigcommerce/cornerstone) theme.
```

**Review by uwikaiddi — CHANGES_REQUESTED** [review]

**Review by uwikaiddi — APPROVED** [review]

**Pull request merged**